backend/emailSystem.py

```diff
@@ -0,0 +1,34 @@
+import smtplib, ssl
+
+port = 465 #For SSL
+password = ""#input("password: ") #we can add the actual password here instead
```

Comment:
Let's add 2FA to the account so that it's safe

Reply:
Steam already ahs 2FA and bc we are using SSO we r covered